fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java

```diff
@@ -764,6 +810,25 @@ private void checkIfOutputAliasNameDuplicatedForGroupBy(Collection<Expression> e
         }
     }
 
+    private boolean hasAggregateFunction(Expression expression, FunctionRegistry functionRegistry) {
+        return expression.anyMatch(expr -> {
+            if (expr instanceof AggregateFunction) {
```

Comment:
only slots in agg's children bind child's output first, for example:
```sql
select abs(sum(c1)) as c1 from t having sum(c2) + c1  > 0
```
in having, `c2` under `sum` bind with agg's child, but `c1` bind with agg's output first

Reply:
crazy logic